client/src/modules-data.json

```diff
@@ -0,0 +1,176 @@
+{
+  "modules": [
+    {
+      "id": 1,
+      "module_name": "Html-Css",
+      "topics": [
+        {
+          "topic_id": 1,
```

Comment:
suggestion: we are using to naming conventions here. Can we try to pick one and enforce it for the entire project 
either 
camel case: e.g. `camelCase` 
underscore: e.g. `topic_id`

Reply:
I have fixed the text and now it uses the camelCase convention throughout. The issue has been successfully resolved.